development/Dockerfile-fakenos

```
@@ -0,0 +1,35 @@
+# Fakenos dockerfile for local development
+
+# Accepts a desired Python version as build argument, default to 3.12
+ARG PYTHON_VER="3.12"
+
+FROM docker.io/python:${PYTHON_VER}-slim-bookworm
```

```suggestion
FROM docker.io/python:${PYTHON_VER}-slim
```
I used bookworm for ltm. This one should be using trixie